tests/test_scrape.py

```diff
@@ -42,6 +42,10 @@ def test_get_images(self):
                   '~text?txtsize=15&txt=image1&w=120&h=120')]
             scrape.get_images('url', directory, 1)
             self.assertEqual(os.listdir(directory), ['filename.png'])
+            try:
+                os.remove('filename.png')
```

Comment:
We're working on a temporary directory here, bro. Just removing the directory soon as it finishes.